kinogoteborg/src/app/booking/[movieID]/page.jsx

```diff
@@ -72,9 +106,24 @@ export default function Page() {
         </div>
 
         <div className="md:col-start-4 md:row-start-8 border justify-center items-center grid">
-          <Button onClick={() => setBookNow(true)} className={"w-[10em]"}>
-            Book Now
-          </Button>
+          {!noSeatsBooked && (
+            <Button
```

Comment:
I would suggest to make the button stand out a bit more its quite pale at the moment